pkg/controller/actuator.go

```diff
@@ -667,3 +671,21 @@ func (a *actuator) findIstioNamespaceForExtension(
 
 	return deployments.Items[0].Namespace, gw.Spec.Selector, nil
 }
+
+func (a *actuator) findDefaultIstioLabels(
+	ctx context.Context,
+) (
+	istioLabels map[string]string,
+	err error,
+) {
+	gwl := istionetworkv1beta1.GatewayList{}
```

Comment:
Can we fetch the `nginx-ingress-controller` explicitly instead of a list?
This would make the extension resistant to additional `Gateways` in the `garden` namespace for different purposes?

Reply:
Actually, I thought we were more resistent this way, assuming that `Gateways` in the `garden` namespace would be exposed via the default `istio-ingressgateway` and also should be protected by this `EnvoyFilter`. So, that if we have `kube-apiserver-ingress`, but for some reason not `nginx-ingress-controller` the detection would sill work.
WDYT?

Reply:
Changed to fetch the `nginx-ingress-controller` explicitly, as discussed in a call.